haystack/nodes/retriever/sparse.py

```diff
@@ -6,17 +6,18 @@
 from sklearn.feature_extraction.text import TfidfVectorizer
 
 from haystack.schema import Document
-from haystack.document_stores import BaseDocumentStore, ElasticsearchDocumentStore
 from haystack.nodes.retriever import BaseRetriever
 
+from haystack.document_stores import BaseDocumentStore
+
 
 logger = logging.getLogger(__name__)
 
 
 class ElasticsearchRetriever(BaseRetriever):
-    def __init__(self, document_store: ElasticsearchDocumentStore, top_k: int = 10, custom_query: str = None):
+    def __init__(self, document_store: "ElasticsearchDocumentStore", top_k: int = 10, custom_query: str = None):  # type: ignore
```

Comment:
Mmm that looks a little weird. But it'll be solved with DCDocumentStore's introduction of `KeywordDocumentStore` anyway.

Reply:
Yes! I should open a small issue not to forget about it. Or did you already tackle it in the main one?

Reply:
Yes is already solved and https://github.com/deepset-ai/haystack/pull/1991 is good to be merged tomorrow.